### ISSUE TITLE
getting errors running the MHS35-show

### ISSUE THREAD

**Issue by madmax828** [lcdgogo/LCD-show#2]
Followed the instructions and ran into these issues. This is my current install for RaspberryPi. I haven't started to try to problem solve these issues yet. But figured i would post this and see if anyone else gets this error when following the directions provided?

Linux raspberrypi 6.6.20+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.20-1+rpt1 (2024-03-07) aarch64 GNU/Linux

/home/pi/LCD-show/rpi-fbcp/main.c:9:10: fatal error: bcm_host.h: No such file or directory
    9 | #include <bcm_host.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/fbcp.dir/build.make:76: CMakeFiles/fbcp.dir/main.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/fbcp.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
install: cannot stat 'fbcp': No such file or directory
(2024-03-07)
./MIS35-show: line 71: test: (2024-03-07): integer expression expected
need to update touch configuration
Selecting previously unselected package xserver-xorg-input-evdev:armhf.
(Reading database ... 130673 files and directories currently installed.)
Preparing to unpack .../xserver-xorg-input-evdev_1%3a2.10.6-1+b1_armhf.deb ...
Unpacking xserver-xorg-input-evdev:armhf (1:2.10.6-1+b1) ...
Processing triggers for man-db (2.11.2-2) ...
dpkg: dependency problems prevent configuration of xserver-xorg-input-evdev:armhf:
 xserver-xorg-input-evdev:armhf depends on libc6 (>= 2.7).
 xserver-xorg-input-evdev:armhf depends on libevdev2 (>= 1.2.2+dfsg-1~).
 xserver-xorg-input-evdev:armhf depends on libmtdev1 (>= 1.1.0).
 xserver-xorg-input-evdev:armhf depends on libudev1 (>= 183).
 xserver-xorg-input-evdev:armhf depends on xorg-input-abi-24.
 xserver-xorg-input-evdev:armhf depends on xserver-xorg-core (>= 2:1.18.99.901).

dpkg: error processing package xserver-xorg-input-evdev:armhf (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 xserver-xorg-input-evdev:armhf
reboot now

**Comment by madmax828** [comment]
Fixed the issues with the following...

apt-get install libraspberrypi-dev

**Comment by madmax828** [comment]
Now another issue... the screen is still blank and nothing is showing up but a blank backlit screen the entire time from boot. Not sure what the issue is since everything seemed to compile and drivers installed fine this time. any clue as to why it isn't working at all?

**Comment by lcdgogo** [comment]
Dear:
You can refer to the following link
http://www.lcdwiki.com/MHS-3.5inch_RPi_Display.






| |
jlttek
|
|
***@***.***
|
---- Replied Message ----
| From | ***@***.***> |
| Date | 4/21/2024 13:20 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | Re: [lcdgogo/LCD-show] getting errors running the MHS35-show (Issue #2) |

Ok i fixed the issue... but the screen is still blank and nothing is showing up but a blank backlit screen the entire time from boot. Not sure what the issue is since everything seemed to compile and drivers installed fine this time. any clue as to why it isn't working at all?

**Comment by abelyuan00** [comment]
I got exactly the same problem as you, but after installed libraspberrypi-dev library and manually installed dependencies by running `sudo apt-get install xserver-xorg-input-evdev:armhf libevdev2:armhf libmtdev1:armhf libudev1:armhf xorg-input-abi-24:armhf xserver-xorg-core:armhf --force-yes` and re-run all the command again, I got the console showing on my monitor (Not the os UI, I wander is it meant to be that case or I got new errors )
